tsstats/utils.py

```diff
@@ -51,7 +51,7 @@ def filter_threshold(clients, threshold):
     return list(filter(lambda c: c[1] > threshold, clients))
 
 
-def flip_ident_map_list(identmap):
+def flip_pretty_ident_map(identmap):
```

Comment:
I'm still not sure about the function-name :/
`flip` doesn't sound *right* to me here, what do you think about `transform`?